pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -148,6 +148,7 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.PodInfraContainerImage = imageTemplate.ExpandOrDie("pod")
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 	server.MaxPods = 110
+	server.SerializeImagePulls = false // disable serial image pulls by default
```

Comment:
This only affects when you're running the kubelet that is a symlink to /usr/bin/origin. Also need to update the appropriate origin files too. @liggitt which ones make the most sense - node_options.go? node_args.go?